<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMajor</class>
 <widget class="QWidget" name="FormMajor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	
	background-color: #ffffff;
}

QToolButton {
border: none;
border-radius: 3px;
background-color: #ffffff;
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:9pt;
font-weight: bold;
color:black;
}

QToolButton:hover {
border: none;
border-radius: 3px;
background-color: #EBF8FF;
min-width: 80px;
font-family: &quot;Microsoft YaHei&quot;;
font-size:9pt;
font-weight: bold;
color:gray;
}

QTableView,QTreeView
{
	border:none;
}

QTreeView
{
	border-right:2px solid #3E75FF;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widgetTop" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="tBtnEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  编辑</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zedit.png</normaloff>:/res/icon/zedit.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="tBtnRefresh">
        <property name="text">
         <string>  刷新</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zrefresh.png</normaloff>:/res/icon/zrefresh.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="tBtnAdd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  添加</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zadd.png</normaloff>:/res/icon/zadd.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="tBtnDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>  删除</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/zdelete.png</normaloff>:/res/icon/zdelete.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <widget class="QToolButton" name="tBtnShowAll">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>显示全部</string>
        </property>
        <property name="icon">
         <iconset resource="../icon.qrc">
          <normaloff>:/res/icon/all.png</normaloff>:/res/icon/all.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string> 学院名称</string>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/res/icon/dir1.png</normaloff>:/res/icon/dir1.png</iconset>
       </property>
      </column>
     </widget>
     <widget class="QTableView" name="tableView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
